proto/ibc/core/channel/v1/tx.proto

```diff
@@ -71,6 +71,8 @@ enum ResponseResultType {
   RESPONSE_RESULT_TYPE_NOOP = 1 [(gogoproto.enumvalue_customname) = "NOOP"];
   // The message was executed successfully
   RESPONSE_RESULT_TYPE_SUCCESS = 2 [(gogoproto.enumvalue_customname) = "SUCCESS"];
+  // The message was executed unsuccessfully
+  RESPONSE_RESULT_TYPE_FAILED = 3 [(gogoproto.enumvalue_customname) = "FAILED"];
```

Comment:
I wonder if this should be `FAILURE` instead of `FAILED`... I'm just thinking if it were `FAILED` then I would expect `SUCCESS` to rather be `SUCCEEDED`

what do ya'll think?